scripts/typegen/main.go

```diff
@@ -225,6 +230,34 @@ func generateRbacObjects(templateSource string) ([]byte, error) {
 		"actionsList": func() []ActionDetails {
 			return actionList
 		},
+		"actionsOf": func(d Definition) []string {
+			// Extract and sort action string keys for deterministic output.
+			list := make([]string, 0, len(d.Actions))
+			for a := range d.Actions {
+				list = append(list, string(a))
+			}
+			slices.Sort(list)
+			return list
+		},
+		"allCaseList": func(defs []Definition) string {
+			// Build a multi-line comma-separated list of all scope constants (including builtins)
+			// suitable for use in a `case ...:` clause, without a trailing comma.
+			var names []string
+			// Builtins first
+			names = append(names, "ScopeAll", "ScopeApplicationConnect", "ScopeNoUserData")
```

Comment:
This is really unfortunate. At the very least, can we add a comment to the generated `Valid` function that these built ins have to be manually added here?

Or can we add something to export the built ins and have this code reference that?